src/main/java/ladder/domain/Command.java

```diff
@@ -0,0 +1,14 @@
+package ladder.domain;
+
+public enum Command {
+    ALL("all"), QUIT("quit");
+    private String value;
```

Comment:
```suggestion
    ALL("all"), QUIT("quit");
    
    private String value;
```

개행을 통해 가독성을 높일 수 있습니다.